common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -67,6 +70,8 @@ public Overlay(
 
     public abstract void render(PoseStack poseStack, float partialTicks, Window window);
 
+    public abstract void renderPreview(PoseStack poseStack, float partialTicks, Window window);
```

Comment:
I think it's a bit sad that we need to have this for all Overlays. :( Can we not just make a default implementation that just shows the overlay title? Otherwise it increases the burden to implement an Overlay. Or, can we not use the *real* render function for previews?

Reply:
If you agree with me making renderPreview just call render, and still have it to be overridden? Let's say the quest info overlay or the game update overlay needs a separate preview, and something like the health bar overlay doesn't.

Reply:
Yes, that sounds better. I can see how you are worried that some overlays may not be visible, and so you want to feed them dummy data. But it also feels a bit odd to make it a separate render function, when it really is like a "get your data from somewhere else". I can't see how we can implement that in a nice way, though.